elyra/pipeline/processor_kfp.py

```diff
@@ -145,8 +145,10 @@ def process(self, pipeline):
             # will be stored
             cos_directory = f'{pipeline_name}-{timestamp}'
 
-        except MaxRetryError as ex:
-            raise RuntimeError('Error connecting to pipeline server {}'.format(api_endpoint)) from ex
+        except requests.exceptions.ConnectionError as ce:
+            raise RuntimeError('Error connecting to pipeline server {}'.format(api_endpoint)) from ce
```

Comment:
Could you please add an user action similar to [what we already have here](https://github.com/elyra-ai/elyra/blob/007cf247dc42082fb7a6cf0c838f241d2cd4b072/elyra/pipeline/processor_kfp.py#L116)

Reply:
With that change, this will be what's produced:
![image](https://user-images.githubusercontent.com/22599560/127406647-06ef0680-8a7d-41c5-8c0c-1d3b038e031c.png)

Is that okay?

Reply:
Yep!